### DISCUSSION TITLE
How set up DEFAULT value for CuratorPicker::make('logo')

### DISCUSSION THREAD

**Discussion by oneplusone-solutions** [awcodes/filament-curator#216] [Q&A] [answered]
default method not working
```
CuratorPicker::make('logo')
                        ->default( 1 ),
```
```
->afterStateHydrated( function ( CuratorPicker $component, $state ) {
              if( !$state ){
                $component->state( 1 );
              }
          })
```
Return an error
"count(): Argument #1 ($value) must be of type Countable|array, int given"
`$component->state( 1 );`
Return an error
"Trying to access array offset on value of type int"

Please help! 
How to set default value???

**Comment by awcodes** [comment, marked as answer]
Make it an array. [1]

**Comment by awcodes** [comment]
Also don't override afterStateHydrated. There's a lot going on there that you will lose functionality wise if you don't copy it over.

**Comment by oneplusone-solutions** [comment]
Yes Array is working
`
->default( [1] )

->acceptedFileTypes( ['image/svg+xml'] )

//                        ->afterStateHydrated( function ( CuratorPicker $component, $state ) {

//                            if( !$state ){

////                                $component->state( [16] );

//                            }

//                        })`

Thanks a lot!
